### DISCUSSION TITLE
- New dev-bnb-nf4-v2 steps 👻

### DISCUSSION THREAD

**Discussion by danilomaiaweb** [lllyasviel/stable-diffusion-webui-forge#1119] [General]
I was testing a merged version of schnell+dev and was getting good results with only 4 steps and an average of 18s to generate each image.

Now I'm testing this new version dev-bnb-nf4-v2 and I only get reasonably good images with 10 steps, which takes up to 50s to generate the image.

A significant increase in generation time.

How many steps are you using this new version to get a reasonable image?

My setup is:
Windows 11 Pro
AMD64 Rayzen 5 3500X
RTX 3050 8GB
32GB RAM
Latest Forge version f2.0.1v1.10.1-previous-252-g00f1cd36
Python 3.10.6
Torch 2.3.1+cu121
Cuda 12.1
Cudnn 8907
Diffusers 0.29.2
Transformers 4.44.0

Please leave your comments
How many steps are you using with dev-bnb-nf4-v2

Thank in advance
Best regards
Brazil

**Comment by pflky** [comment]
You would do 20 steps or more. It's not a low step model. Using the beta scheduler can help too.

**Reply by danilomaiaweb** [reply]
> You would do 20 steps or more. It's not a low step model. Using the beta scheduler can help too.

Please see the model merged with 6 steps:
![image](https://github.com/user-attachments/assets/cf3cfee8-032a-4f6b-998e-c0c339ab9b6b)

![image](https://github.com/user-attachments/assets/78202383-0064-485b-bab7-6f79ba13d9be)

The result:
![00266-14-08-2024 - 2607611297](https://github.com/user-attachments/assets/ae747161-e6d9-420a-af40-5e8ac8e64038)

**Comment by danilomaiaweb** [comment]
With 4 steps in model FLUX1-SchnellDev-Merged-fp8-4step.safetensors:
![image](https://github.com/user-attachments/assets/681f0010-8899-41c9-b4d8-c9421fd02e29)

Generation in 24s:
![image](https://github.com/user-attachments/assets/5068be92-7b0a-4b20-b021-dc10fb77c500)

Result:
![00002-15-08-2024 - 483473153](https://github.com/user-attachments/assets/6bc1e9e8-c0e5-4b09-80dd-a7a1b2b49436)
